### DISCUSSION TITLE
Welcome to skimmer_webapp Discussions!

### DISCUSSION THREAD

**Discussion by cwhelchel** [cwhelchel/skimmer_webapp#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dj1yfk** [comment]
Hi Cainan,

introduction: Fabian, DJ5CW (ex DJ1YFK, thus my github user name) from Munich, Germany. In my (very) late 30s, ham since 1997, interested in anything CW (which includes SKCC; I am slowly working my way to Tx2 right now) and occasionally doing some ham related coding (Perl, PHP, Python, C, Go...)  :-)

Nice project!

Not directly related, but maybe of some interest: A few weeks ago I looked into how the SKCC Sked Page is implemented and wrote a bridge to IRC. Ever since I am basically lurking on the Sked Page 24/7 since an IRC client is always running here anway :-) 
Here's the project: https://git.fkurz.net/dj1yfk/skcc-irc (also mirrored to github:  https://github.com/dj1yfk/skcc-irc)

I was thinking about including SKCC Skimmer into this as well, so the "skcc" bot in the channel would provide spots (configurable verbosity).

Maybe you would be interested in the implementation of the Chat and re-use it to include it into your web app. Maybe not :-)

73
Fabian

**Reply by cwhelchel** [reply]
Very cool Fabian. Thanks for your link. I don't use IRC much but I like the thought of adding the sked chat. I'll have to take a closer look at your project. Thank you!

I'm also in my _VERY_ late 30s..... I'll be 40 this year. 😭 
But i've only just become a ham.
